packages/theme_tailor/lib/src/generator/theme_tailor_generator.dart

```diff
@@ -40,36 +41,66 @@ class ThemeTailorGenerator extends GeneratorForAnnotation<Tailor> {
         todo: 'Move @Tailor annotation above `class`',
       );
     }
-    final library = element.library;
+    final resolver = buildStep.resolver;
+    final library = await resolver
+        .libraryFor(await resolver.assetIdForElement(element.library));
+
```

Comment:
Is this change required? (I didn't catch why it is necessary)

Reply:
I tested the code generation a lot of times, and once in a while I was getting exception originating from this line, but I don't really remember what that an exception was because I worked on this long time ago. I will remove this change for now since it is not really related to constant themes.